curation-migration-datasync/config.ts

```diff
@@ -22,13 +22,25 @@ const config = {
       curatedRecIdHashKey:
         process.env.CURATION_MIGRATION_TABLE_HASH_KEY || 'curatedRecId',
     },
+    eventRule: {
+      source: 'curation-migration-datasync',
+    },
   },
   db: {
     secretId: process.env.DATABASE_SECRET_ID || '',
     port: process.env.DATABASE_PORT || '3310',
     dbName: process.env.DATABASE || 'readitla_ril-tmp',
     tz: process.env.DATABASE_TZ || 'US/Central',
   },
+  tables: {
+    curated_feed_prospects: 'curated_feed_prospects',
+    curated_feed_queued_items: 'curated_feed_queued_items',
+    curated_feed_items: 'curated_feed_items',
+    curated_feed_topics: 'curated_feed_topics',
+    tile_source: 'tile_source',
+    syndicated_articles: 'syndicated_articles',
```

Comment:
could we use `camelCase` for the keys

Reply:
ya, i was thinking about that - but kept both the table name and column name in snake case as its exactly matching the db schema

Reply:
Having the table names in a config serves as a layer of abstraction so we don't have to concern ourselves with the actual db table names. I don't think we need to keep the case style or even the names the same but rather use a standard that works for the codebase.